crates/hotshot/src/traits/networking/push_cdn_network.rs

```diff
@@ -153,6 +153,18 @@ pub struct PushCdnNetwork<TYPES: NodeType> {
     is_paused: Arc<AtomicBool>,
 }
 
+/// The enum for the topics we can subscribe to in the Push CDN
+#[repr(u8)]
+pub enum Topics {
+    /// The global topic
+    Global = 0,
+    /// The DA topic
+    DA = 1,
+}
```

Comment:
Very minor nit, but should this be `Topic` (singular)?

Reply:
Fixed, thanks!